### ISSUE TITLE
Add support for auto-play

### ISSUE THREAD

**Issue by austinhinderer** [austinhinderer/Player#29]

**Comment by austinhinderer** [comment]
I'm going to look into this

**Comment by toastynerd** [comment]
So is this autoplay like I have five videos and it will play one after the next or auto play like I just opened the page and it's playing? (please let it be the first, please let it be the first)

**Comment by swansong** [comment]
I think we're going to need support for both, but neither of them should be on by default. I'll open an issue for playlist autoplay next.

**Comment by austinhinderer** [comment]
Yeah I was talking about the latter, but we will want the former as well.
